### PR TITLE
OOP Implementation, strict PEP8 coherence, added gitignore

### PR DESCRIPTION
**Pull request by EclectickMedia**
Changes naming convention for variables and methods from camelcase to underline separated styles. This is recommended by PEP8, but is largely style.

### CODE REVIEW

**Comment by EclectickMedia** [comment]
Added a setup.py file in 5a11d84

File: main.py [hunk 1]
```diff
@@ -87,201 +87,193 @@ def allAsCircles(item):
...
+        draw_as_circles = True
+
+        # Number of dots to generate
+        dot_count = 5000
+
+        # Developer variables, do not touch these
+        origin_x = self.width / 2  # Center of the screen X
+        origin_y = self.height / 2  # Center of the screen Y
+        self.points = []  # Array will store all points on screen
+        self.epochTime = time.clock()
```
Comment (EclectickMedia) [inline main.py]: `self.epochTime` should be `self.epoch_time` to stay coherent with PEP8

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+with open('README.md') as f:
+    readme = f.read()
+
+# TODO Add author email.
+setup(
+    name='thetriangle',
+    description='Generate serpinski triangles using cool maths',
+    long_description=str(readme),
+    license='MIT',
+    author='Justyn Chaykowski',
```
Comment (EclectickMedia) [inline setup.py:12]: To add author email, we just need: `author_email='XXXX@gmail.com`

File: main.py (hunk 1 above)
Comment (EclectickMedia) [reply main.py]: Fixed in a635183

**Review by frawst — APPROVED** [review]

**Pull request merged**